packages/ui/src/views/profile-settings/components/profile-settings-keys-list.tsx

```diff
@@ -0,0 +1,71 @@
+import { Icon, Table, TableBody, TableCell, TableHead, TableHeader, TableRow, Text } from '@/components'
+import { timeAgo } from '@utils/utils'
+
+import { KeysList } from '../types'
+
+interface PageProps {
+  publicKeys: KeysList[]
+  openAlertDeleteDialog: (params: { identifier: string; type: string }) => void
+}
+
+export const ProfileKeysList: React.FC<PageProps> = ({ publicKeys, openAlertDeleteDialog }) => {
+  return (
+    <Table variant="asStackedList">
```

Comment:
quick question would we be able to use a stackedlist or do we need a table?

Reply:
`StackedList` only allows rendering a single column with `left` and `right` props. This has multiple columns.